{{cookiecutter.repo_name}}/{{cookiecutter.project_name}}/routes/example/crud.py

```diff
@@ -8,11 +8,7 @@
 from microcosm_flask.operations import Operation
 from microcosm_postgres.context import transactional
 
-from {{ cookiecutter.project_name }}.resources.example_resources import (
-    ExampleSchema,
-    NewExampleSchema,
-    SearchExampleSchema,
-)
+from {{ cookiecutter.project_name }}.resources.example_resources import ExampleSchema, NewExampleSchema, SearchExampleSchema
```

Comment:
Huh, thought this would have extended past the 99 char mark. Doesn't matter that much either way.